### PR TITLE
Upgrade to OpenMapTiles 3.9 release

### PR DESCRIPTION
**Pull request by nlehuby**
- Add labels to park layer #525, #521 
- Add multilingual names for streets  #523 
- Add piers and docks #524, #529 
- Add indoors fields for POIs #435
- Add type of sport as subclass of pitch #532
- Add aquariums #452
- Add beaches #418

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ services:
      - ./build:/mapping
      - cache:/cache
   import-sql:
-    image: "openmaptiles/import-sql:0.7"
+    image: "openmaptiles/import-sql:0.8"
```
Comment (amatissart) [inline docker-compose.yml:71]: Note: we will need to update this dependency in kartotherian_config

File: layers/building/building.sql [hunk 2]
```diff
@@ -56,31 +60,37 @@ CREATE OR REPLACE VIEW osm_all_buildings AS (
...
+                  COALESCE(nullif(as_numeric(obp.height),-1),nullif(as_numeric(obp.buildingheight),-1)) as height,
+                  COALESCE(nullif(as_numeric(obp.min_height),-1),nullif(as_numeric(obp.buildingmin_height),-1)) as min_height,
+                  COALESCE(nullif(as_numeric(obp.levels),-1),nullif(as_numeric(obp.buildinglevels),-1)) as levels,
+                  COALESCE(nullif(as_numeric(obp.min_level),-1),nullif(as_numeric(obp.buildingmin_level),-1)) as min_level,
+                  CASE WHEN obr.role='outline' THEN TRUE ELSE FALSE END as hide_3d
          FROM
-         osm_building_polygon WHERE osm_id >= -1e17
+         osm_building_polygon obp
+           LEFT JOIN osm_building_relation obr ON (obr.member = obp.osm_id)
+         WHERE obp.osm_id >= 0
```
Comment (amatissart) [inline layers/building/building.sql]: This breaks building polygons.  
As we use "single_id_space", a way will have an osm_id between -1e17 and 0.
So we need to keep the filter `osm_id >= -1e17` (Or find a cleaner configuration to handle that difference ?)

File: layers/poi/poi.yaml [hunk 3]
```diff
@@ -41,12 +41,23 @@ layer:
...
+          Original value of [`indoor`](http://wiki.openstreetmap.org/wiki/Key:indoor) tag.
+      values:
+      - 1
   datasource:
     geometry_field: geometry
     key_field: osm_id
     key_field_as_attribute: no
     srid: 900913
-    query: (SELECT osm_id, geometry, global_id, name, name_en, name_de, {name_languages}, class, subclass, agg_stop, rank, tags::json::text as tags FROM layer_poi(!bbox!, z(!scale_denominator!), !pixel_width!)) AS t
+    query: (SELECT osm_id, geometry, global_id, name, name_en, name_de, {name_languages}, class, subclass, agg_stop, layer, level, indoor, rank, tags::json::text as tags FROM layer_poi(!bbox!, z(!scale_denominator!), !pixel_width!)) AS t
```
Comment (amatissart) [inline layers/poi/poi.yaml]: Is that not a good opportunity to remove `tags` from POI tiles right now ?

**Review by amatissart — COMMENTED** [review]
Besides merge issues, there are also some differences that are due to an older version of openmaptiles-tools. (Please use the latest version, 0.9.2)

**Comment by nlehuby** [comment]
Add performances optimizations:
* remove all tags in `poi` tiles
* remove ~40 unused languages (but add French missing ones)
* remove layers not used in style

File: layers/poi/layer.sql [hunk 4]
```diff
@@ -36,9 +36,14 @@ RETURNS TABLE(osm_id bigint, global_id text, geometry geometry, name text, name_
                 THEN NULLIF(information, '')
             WHEN subclass = 'place_of_worship'
                     THEN NULLIF(religion, '')
+            WHEN subclass = 'pitch'
+                    THEN NULLIF(sport, '')
```
Comment (amatissart) [inline layers/poi/layer.sql:40]: These subclass rules are hidden in the tile query, and can't be retrieved by Fafnir. (And as a consequence icons used on the map will not match with UI).  
Should we define a new function poi_subclass, similar to `poi_class` ? (Not so convincing: the signature of this function would change when new tags are used...).

As far as i can see, this is the only remaining problem (but not directly related to this PR).

**Comment by nlehuby** [comment]
Here are a few screenshots:
layer park with a name:
capture_park_name

some pier:
man_made pier

some aquarium:
capture du 2018-12-13 11-26-04

a street with a `name:fr` (they only had `name:en` and `name:de` before):
capture_name_streets

some POI with an indoor tag (`level`) and no tags:
capture_poi_indoor

a pitch with a sport subclass:
capture_sports

**Review by amatissart — APPROVED** [review]

**Pull request merged**